libs/hmpb/src/ballot_templates/nh_ballot_template.tsx

```diff
@@ -218,6 +243,374 @@ function BallotPageFrame({
   );
 }
 
+function CandidateContest({
```

Comment:
Almost all of this is copy + pasted from `vx_default_ballot_template` except for styling and DOM tree position of `<Bubble>` and `<BubbleWrapper>` components, noted below